components/article/client/image_set.coffee

```diff
@@ -22,6 +28,22 @@ module.exports = class ImageSetView extends Backbone.View
     @__postRender__() unless @__postRendered__
     this
 
+  setupFollowButtons: ->
+    @artists = []
+    $('.artist-follow').each (i, artist) =>
```

Comment:
@kanaabe made a great suggestion to add a template or client test. This is minor but while you are in here you could remove the `i` variable in the each block sense it's not being used.